docs/introduction/running-acceptance-tests.md

```diff
@@ -22,6 +22,13 @@ php phing tests-acceptance
 *Note: In this step you were using Phing target `tests-acceptance`.
 More information about what Phing targets are and how they work can be found in [Console Commands for Application Management (Phing Targets)](/docs/introduction/console-commands-for-application-management-phing-targets.md)*
 
+### How to watch what is going on in the selenium browser
+1. From your local machine, connect to the remote desktop on `vnc://127.0.0.1:5900`
```

Comment:
```suggestion
1. From your local machine, connect to the remote desktop on `vnc://127.0.0.1:4400`
```

Reply:
That's not the right port, doesn't connect to a remote desktop, see https://github.com/shopsys/shopsys/pull/1251#issuecomment-517697351

![image](https://user-images.githubusercontent.com/10008612/62372996-203ad280-b539-11e9-89e6-42e7dc4d1cb4.png)

Reply:
Oh, ok there needs to be that port set in docker-compose